### ISSUE TITLE
Parameter scalePoint values are not sorted in UI drop down

### ISSUE THREAD

**Issue by SpotlightKid** [OpenMusicKontrollers/moony.lv2#14]
Example: I define a parameter is defined like this:

```lua
local filter_chan = Parameter {
  [RDFS.label] = 'Filter channel',
  [RDFS.comment] = 'Set MIDI channel to which this filter is applied.',
  [RDFS.range] = Atom.Int,
  [RDF.value] = -1,
  [LV2.minimum] = -1,
  [LV2.maximum] = 15,
  [LV2.scalePoint] = {
    ["All Channels"] = -1,
    ["Channel 1"] = 0,
    ["Channel 2"] = 1,
    [...]
    ["Channel 15"] = 14,
    ["Channel 16"] = 15,
  }
}
```

The UI will show a drop down with the correct values but they are out of order.

![Bildschirmfoto_2020-09-09_00-05-32](https://user-images.githubusercontent.com/5719151/92532378-7431e380-f230-11ea-8efc-a7fc820d0f06.png)

**Comment by ventosus** [comment]
Thanks for reporting, can reproduce this.

**Comment by ventosus** [comment]
fixed in 1da6ee65f5f7f07dba8ca27a7f1e03ba6410d36b

**Comment by ventosus** [comment]
ah, no, it is not fixed, yet

**Comment by ventosus** [comment]
fixed in 0553d9681d0b905750e2ad6009c4b457aad1f582
